target/linux/ar71xx/files/arch/mips/ath79/mach-cf-e316n-v2.c

```diff
@@ -130,6 +131,58 @@ static struct gpio_keys_button cf_e320n_v2_gpio_keys[] __initdata = {
 	},
 };
 
+/* CF-E355AC */
+#define CF_E355AC_GPIO_BTN_RESET        17
+
+#define CF_E355AC_GPIO_XWDT_TRIGGER     13
+
+#define CF_E355AC_GPIO_LED_WAN          2
+#define CF_E355AC_GPIO_LED_LAN          3
+#define CF_E355AC_GPIO_LED_WLAN         0
+
+#define CF_E355AC_GPIO_LED_EXT_WAN              16
+#define CF_E355AC_GPIO_LED_EXT_LAN              14
+#define CF_E355AC_GPIO_LED_EXT_WLAN             11
+
+static struct gpio_led cf_e355ac_leds_gpio[] __initdata = {
+	{
+		.name           = "cf-e355ac:red:wan",
+		.gpio           = CF_E355AC_GPIO_LED_WAN,
+		.active_low     = 0,
+	}, {
+		.name           = "cf-e355ac:green:lan",
+		.gpio           = CF_E355AC_GPIO_LED_LAN,
+		.active_low     = 0,
+	}, {
+		.name           = "cf-e355ac:blue:wlan",
+		.gpio           = CF_E355AC_GPIO_LED_WLAN,
+		.active_low     = 0,
+	}, {
+		.name           = "cf-e355ac:blue:ext_wan",
```

Comment:
I do not know exactly, I get from the manufacturer, when I clarify, I'll create a new commit.